src/components/EventManagement/EventAttendance/AttendanceList/AttendedEventList.tsx

```
@@ -48,31 +51,33 @@ const AttendedEventList: React.FC<Partial<InterfaceEvent>> = ({ id }) => {
     <React.Fragment>
-      <Table className="bg-primary" aria-label="Attended events list">
+      <Table
+        className="bg-primary"
+        aria-label={t('attendedEventsList') as string}
+      >
         <TableBody className="bg-primary">
           {event && (
             <TableRow
               key={event.id}
               className="bg-white rounded"
               role="row"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<tr>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)